src/plone/restapi/tests/http-examples/types_document.resp

```diff
@@ -97,7 +97,7 @@ Content-Type: application/json+schema
       }
     }, 
     "author_email": {
-      "behavior": "plone.dexterity.schema.generated.plone_0_Document", 
+      "behavior": "plone.dexterity.schema.generated.plone_5_1616411693_2_966875_0_Document", 
```

Comment:
@sneridagh seems we need to pin plone.dexterity to 2.9.8 to avoid this (see https://github.com/plone/plone.restapi/issues/1096)